pom.xml

```diff
@@ -49,27 +50,27 @@
             <version>1.3.78</version>
         </dependency -->
         <dependency>
-	    <groupId>com.github.jagrosh</groupId>
-	    <artifactId>lavaplayer</artifactId>
-	    <version>jmusicbot-SNAPSHOT</version>
-	</dependency>
```

Comment:
Revert to our fork for now, if possible. I'm guessing this isn't compatible with JDA 5 either, so jagrosh needs to say if we want to switch to a fork or want to maintain our own fork.

Reply:
Edited :)